code/frontend/package.json

```diff
@@ -7,7 +7,7 @@
         "dev": "tsc && vite",
         "build": "tsc && vite build",
         "watch": "tsc && vite build --watch",
-        "test": "mocha"
+        "test": "vitest run"
```

Comment:
I realized that vitest is much more lightweight as a unit test framework when testing in typescript. Plus it allows for super-easy mocking of environment variables.